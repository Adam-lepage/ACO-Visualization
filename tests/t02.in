input2.txt
0
10
20
-1
0.5