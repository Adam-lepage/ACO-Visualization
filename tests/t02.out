The best found solution is of length 13, which used the following path:
1. 0
2. 3
3. 1
4. 4
5. 2
6. 5